### ISSUE TITLE
Focus Indicator on Captcha check box is not visible in High Contrast Black/White themes

### ISSUE THREAD

**Issue by v-vasud** [google/recaptcha#506]
**Issue description**
Focus Indicator on Captcha check box is not visible in High Contrast Black and White themes

**Environment**

 * OS name and version: Windows 10 & 11
 * `google/recaptcha` version: Google recatcha component from Kentico version 11
 * Browser name and version: Edge & Chrome

**Reproducing the issue**
Navigate to High Contrast theme settings and select Black/White in Windows 10 or Desert or Aquatic in high OS versions. And then access the below mentioned URL from Edge or Chrome Browser and navigate onto the check box using tab key

URL: https://www.google.com/recaptcha/api2/demo)

**Comment by rowan-m** [comment]
Repo is just for PHP client issues. This is about the reCAPTCHA service.
